cmd/build.go

```diff
@@ -55,46 +48,52 @@ func Build(ctx context.Context) *cobra.Command {
 			if err := utils.CheckIfDirectory(path); err != nil {
 				return fmt.Errorf("invalid build context: %s", err.Error())
 			}
+			options.Path = path
 
-			if file == "" {
-				file = filepath.Join(path, "Dockerfile")
+			if options.File == "" {
+				options.File = filepath.Join(path, "Dockerfile")
 			}
 
-			if err := utils.CheckIfRegularFile(file); err != nil {
+			if err := utils.CheckIfRegularFile(options.File); err != nil {
 				return fmt.Errorf("invalid Dockerfile: %s", err.Error())
 			}
 
 			buildKitHost, isOktetoCluster, err := build.GetBuildKitHost()
 			if err != nil {
 				return err
 			}
-			log.Information("Running your build in %s...", buildKitHost)
+			if buildKitHost == "" || options.BuildLocally {
+				log.Information("Running your build in docker...")
```

Comment:
"running your build" sounds odd. "Building your image using your local docker daemon" maybe? (not sure what docker displays here)
```suggestion
				log.Information("Running your build in docker...")
```